### ISSUE TITLE
WPI for nullness fails to annotate a multi-field declaration

### ISSUE THREAD

**Issue by msridhar** [typetools/checker-framework#6794]
Input:
```java
class NoExplicitInitMultiField {
  // :: warning: (initialization.field.uninitialized)
  Object f, g;
}
```
WPI for the Nullness Checker does not infer a `@MonotonicNonNull` or `@Nullable` annotation for the declaration.
